sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -83,6 +83,8 @@ internal class AmqpReceiver : TransportReceiver
         /// </summary>
         public override string SessionId { get; protected set; }
 
+        private readonly string _transactionGroup;
```

Comment:
nit:  Any reason to not declare this with the other fields rather than in the middle of the properties?